.github/workflows/shippable_builds.yml

```diff
@@ -261,7 +346,26 @@ jobs:
       RELEASE_TYPE: ${{ needs.dump_config.outputs.releaseType }}
       APP_NAME: ${{ matrix.appName }}
       PACKAGE_FLAVOR: ${{ matrix.packageFlavor || 'default' }}
+      THUNDERBIRD_SHA: ${{ needs.commit_thunderbird.outputs.thunderbird_sha }}
+      K9MAIL_SHA: ${{ needs.commit_k9mail.outputs.k9mail_sha }}
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
```

Comment:
This step doesn't actually use the repository, so we don't need to check it out. It will fail because the `softprops/action-gh-release` action uses `github.sha` instead of your ref.